scripts/get_most_recent_ci_hash.py

```diff
@@ -19,6 +19,24 @@ def gcc_hashes(hash: str, subsequent: bool):
 
     return commits
 
+def get_valid_artifact_hash(hashes:List[str], token: str, artifact_name: str):
```

Comment:
It doesn't make sense for us to have both `get_valid_artifact_hash` and `get_valid_hash`. Feel free to delete `get_valid_hash`.